products/flux2.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Flux
+category: server-app
+tags: cncf
+iconSlug: flux2
+permalink: /flux
+alternate_urls:
+-   /flux2
+-  /fluxcd
+versionCommand: flux version
+releasePolicyLink: https://fluxcd.io/flux/releases/#supported-releases
+changelogTemplate: https://github.com/fluxcd/flux2/releases/tag/v__LATEST__
+activeSupportColumn: false
+releaseDateColumn: true
+eolColumn: Support
+
+auto:
+-   git: https://github.com/fluxcd/flux2.git
```

Comment:
```suggestion
-   git: https://github.com/fluxcd/flux2.git
-   git: https://github.com/fluxcd/flux.git
```

Reply:
We're not exactly keen on pointing people to a Flux version that has been superseded by a new major one for several years now.

Reply:
The main purpose of endoflife.date is to document EOL versions, with the EOL date if it's known. So I still think it would be a valuable information.

Moreover this link is only used by our automation scripts to get the release dates, it's not present on the page. But it will be used by our API so it can answer the question "is version 1.x EOL ?".

Reply:
Flux v1 EOL on Nov 2, 2022. I see no issue with adding this info.

Reply:
Thanks @stefanprodan for the date.